cassandra-bridge/src/main/java/org/apache/cassandra/bridge/CassandraBridge.java

```diff
@@ -200,65 +192,152 @@ public List<CqlField.NativeType> supportedTypes()
 
     // Native
 
-    public abstract CqlField.NativeType ascii();
+    public CqlField.NativeType ascii()
+    {
+        return cassandraTypes().ascii();
+    }
```

Comment:
How about only exposing `cassandraTypes()` from bridge and getting the specific type at the callsites? Maybe it comes in later refactoring?

Reply:
I kept this as a convenience, but also to reduce the PR diff.